composer.json

```
@@ -22,12 +22,12 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
+        "php": ">=7.3",
```

```suggestion
        "php": ">=7.2",
```
Can leave this at 7.2

---

Yeah Sure. for version below below 8.x.